tendrl/node_agent/node_sync/__init__.py

```diff
@@ -54,29 +55,34 @@ def _run(self):
                         priority="error",
                         publisher=NS.publisher_id,
                         payload={"message": "node_sync "
-                                            "os/cpu/memory sync failed: " +
-                                            ex.message,
+                                 "os/cpu/memory sync failed: " +
+                                 ex.message,
                                  "exception": ex}
                     )
                 )
                 NS.node_context = NS.node_context.load()
                 NS.node_context.sync_status = "failed"
                 NS.node_context.last_sync = str(time_utils.now())
-                NS.node_context.save()
+                NS.node_context.save(ttl=_sync_ttl)
 
             sync_disks_thread = gevent.spawn(disk_sync.sync)
             sync_disks_thread.join()
 
             sync_networks_thread = gevent.spawn(network_sync.sync)
             sync_networks_thread.join()
 
+            if "tendrl/monitor" in NS.node_context.tags:
+                update_node_status_thread = gevent.spawn(
+                    node_status_update.update)
```

Comment:
instead of node_status_update.update

call it "check_all_managed_nodes_status.run()"

Reply:
ack